paasta_tools/marathon_serviceinit.py

```diff
@@ -98,19 +95,38 @@ def get_bouncing_status(service, instance, client, job_config):
         return PaastaColors.red("Unknown (count: %s)" % app_count)
 
 
+def get_bouncing_status(service, instance, client, job_config):
+    apps = marathon_tools.get_matching_appids(service, instance, client)
+    app_count = len(apps)
+    bounce_method = job_config.get_bounce_method()
+    return bouncing_status_human(app_count, bounce_method)
+
+
+def desired_state_human(desired_state, instances):
+    if desired_state == 'start' and instances != 0:
+        return PaastaColors.bold('Started')
+    elif desired_state == 'start' and instances == 0:
+        return PaastaColors.bold('Stopped')
+    elif desired_state == 'stop':
+        return PaastaColors.red('Stopped')
+    else:
+        return PaastaColors.red('Unknown (desired_state: %s)' % desired_state)
+
+
+def get_desired_state(desired_state, instances):
+    return desired_state_human(desired_state, instances)
```

Comment:
if this just delegates to the ``desired_state_human`` function, then why have 2 of them at all?